docs/source/conf.py

```diff
@@ -18,7 +18,11 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-sys.path.insert(0, os.path.abspath('../../'))
+sys.path.insert(0, os.path.abspath('../..'))
+
+# Set flag to not fail doc build.
+if 'DO_NOT_CHECK_EXECUTABLE_EXISTENCE' not in os.environ:
```

Comment:
I had added this 'DO_NOT_CHECK_EXECUTABLE_EXISTENCE' for Raven doc only.  This is birdy doc, I do not think you need this here.